tests/html/firstDiscoveryEditor-Test.html

```diff
@@ -27,7 +27,16 @@ <h2 id="qunit-userAgent"></h2>
         <ol id="qunit-tests"></ol>
 
         <!-- Test HTML -->
-        <div id="gpiic-fd">
+        <div id="gpiic-fd-navButtons">
+            <section class="gpiic-fd-prefsEditor"></section>
+
+            <div class="gpiic-fd-navButtons">
```

Comment:
did you mean to have the `gpiic-fd-navButtons` element inside another element with the same class name?